Source/Fuse.Reactive.Expressions/BinaryOperator.uno

```diff
@@ -27,14 +27,18 @@ namespace Fuse.Reactive
 		protected virtual bool IsLeftOptional { get { return false; } }
 		protected virtual bool IsRightOptional { get { return false; } }
 
-		protected virtual object Compute(object left, object right)
+		protected virtual bool Compute(object left, object right, out object result)
```

Comment:
This also breaks public API :/

Reply:
I think I can add the old API and mark it deprecated (they have different overload paraemters so it shouldn't conflict).

Reply:
If you can find a way that works, then that would be great. It's not clear to me how we'd know which version to call, though.

Reply:
mark the old one as deprecated?

Reply:
@yupferris: If you're asking how we do that, the answer is that we use the `[Obsolete]`-attribute in uno.

Reply:
I've pushed the backwards compatible version. It can't issue a compile-time error, but will emit a runtime error about compatibility.